docs/design/snapshotters.md

```diff
@@ -0,0 +1,32 @@
+# Snapshotters Config Option
+
+## Issue
+
+[#506](https://github.com/runfinch/finch/pull/506)
+
+## Supported snapshotters
+
+- [SOCI](https://github.com/awslabs/soci-snapshotter)
+- OverlayFS (default snapshotter of nerdctl/containerd)
+
+## ```snapshotters``` option
+
+Users can use the snapshotters option at ```${HOME}/.finch/finch.yaml``` to set the snapshotters they would like to use.
```

Comment:
nit: suggested above but I see it a few other times, for inline code blocks you can use single backticks \` my code here  \`